pyproject.toml

```diff
@@ -40,6 +40,7 @@ test = [
     "pytest-cov",
     "nbproject-test>=0.4.3",
     "sqlmodel",
+    "pyjwt",
```

Comment:
I think pyjwt needs to be a dependency not a test-dependency! Can you move it into the actual dependencies, please?